.gitignore

```diff
@@ -1 +1,3 @@
 *swp
+*log
```

Comment:
*.log, *.swp?

Reply:
Vagrant leftover log. And my own vim .swp log, so I don't see it when I write git status.

Reply:
I was thinking if *log is not too restrictive, it would "eat" a "catalog" too :)